app/assets/javascripts/live_search.js

```diff
@@ -79,6 +79,9 @@
     this.$emailLink.attr(
       'href', this.emailSignupHref + "?" + $.param(this.state)
     );
+    this.$emailLink.attr(
+      'data-track-action', this.emailSignupHref + "?" + $.param(this.state)
```

Comment:
I assume we don't want this to diverge from `this.$emailLink.attr('href')`? If so maybe we we could extract a variable or reference the `$emailLink` href attribute?

Reply:
I think this URL construction is fundamentally broken as it omits the case where there is an existing query string.  I think I'm going to have to do a bit more on it.  Thanks for the sanity check

Reply:
FYI, this is fixed in https://github.com/alphagov/finder-frontend/pull/832.